config/gadi.config

```diff
@@ -1,38 +1,104 @@
 // profile for the Gadi supercomputer at NCI
-// Before running will need to run: 
-// module load nextflow singularity 
-
-// Gadi specific parameters
 // More info: https://opus.nci.org.au/display/DAE/Nextflow
-params {
-    pbs_account = ''
-    whoami = ''
-}
 
 // Preset use of containers with Singularity
 singularity {
     enabled = true
-    cacheDir = "/scratch/${params.pbs_account}/${params.whoami}/.nextflow/singularity"
+    cacheDir = "/scratch/${params.gadi_account}/${params.whoami}/singularity"
```

Comment:
Confirm `gadi_account` matches parameter description in nextflow.config and README